test/html.test.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const assert = require('assert');
```

Comment:
yassssssss

Reply:
i don't have opinions on assertion libraries, really. for some reason I used `assert` when writing the very first tests because it felt easier than justifying another dependency. Here I'm just being consistent.